src/DbPostgreSQLAdapter/Database.cpp

```diff
@@ -0,0 +1,134 @@
+#include "stdafx.h"
+
+#include "Database.h"
+#include "DbAdapterInterface/ITable.h"
+#include "RecordSet.h"
+#include "Table.h"
+#include "TableRecordSet.h"
+#include "Transaction.h"
+
+#include <libpq-fe.h>
+
+namespace {
+	std::unique_ptr<PGresult, void(*)(PGresult*)> createRAIIPGresult(PGresult* result)
```

Comment:
You have the same function in Connection.cpp but named _makePGResult_. Maybe you could extract this kind of _helper functions__ into a _PostgresHelper_ class with static methods to reuse them

Reply:
I'll move it